substrate/frame/support/src/traits/members.rs

```diff
@@ -137,6 +137,14 @@ impl<A, B, These: ContainsPair<A, B>, Those: ContainsPair<A, B>> ContainsPair<A,
 	}
 }
 
+/// An implementation of [`Contains`] which contains only equal members to `T`.
+pub struct EqualsTo<T>(PhantomData<T>);
+impl<X: PartialEq, T: super::Get<X>> Contains<X> for EqualsTo<T> {
```

Comment:
Why did you rename `Equals` to `EqualsTo`? "Equals to" is not grammatically correct in standard written/spoken English.

Reply:
caramba, reverted, I think I saw some similar functions:
```
pub fn equals_to(&self, other: &Self) -> bool {
...
pub fn canon_equals_to(&self, other: &Self) -> bool {
```